WordPress/src/main/res/layout/theme_activation_bottom_sheet.xml

```diff
@@ -0,0 +1,173 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <include
+        android:id="@+id/bottom_sheet_handle"
+        layout="@layout/bottom_sheet_handle_view"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content" />
+
+    <ImageButton
+        android:id="@+id/close_button"
+        android:layout_width="@dimen/theme_activation_bottom_sheet_close_button_size"
+        android:layout_height="@dimen/theme_activation_bottom_sheet_close_button_size"
+        android:layout_alignParentEnd="true"
+        android:layout_below="@id/bottom_sheet_handle"
+        android:layout_margin="@dimen/margin_extra_large"
+        android:background="@drawable/close_action_sheet_circle_selector"
+        android:contentDescription="@string/label_close_button"
+        android:cropToPadding="true"
+        android:padding="@dimen/theme_activation_bottom_sheet_close_button_padding"
+        android:src="@drawable/ic_close_black_24dp"
+        android:tint="@color/gray_30" />
+
+    <TextView
+        android:id="@+id/theme_name_tv"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_below="@id/close_button"
+        android:layout_centerHorizontal="true"
+        android:fontFamily="@font/libre_baskerville"
```

Comment:
Ideally we want to avoid hardcoding fonts or other attributes in our views and reuse (or create new ones if needed) styles from `styles.xml`. That way we will be more flexible in adjusting the UI to match user preferences. For example we will need to cover dark mode.

<details>
 <summary>Dark mode screenshot</summary>
 
<img width="320" src="https://user-images.githubusercontent.com/304044/125801573-426ae467-7421-4ccc-a645-608fcbb504b2.png"/>
 
</details>

Reply:
Note that since we do not have all the colors for the dark mode at this point we can try reusing the styles from similar screens/controls.

Reply:
Dark mood after applying this suggestion
![Screenshot_1626438421](https://user-images.githubusercontent.com/32029435/125947349-13eac4ae-d8ff-401e-bb88-a2234c6f886c.png)

Reply:
This looks a lot better now. Thank you for making this change 🙇